File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SendDataResult.java
package com.sierrawireless.avphone.task;


public class SendDataResult {
    private boolean error;
    private String lastLog;


    SendDataResult(String lastLog, boolean error) {
        this.error = error;
        this.lastLog = lastLog;
    }

    SendDataResult(String lastLog) {
        this.lastLog = lastLog;
        this.error = false;
    }

    public boolean isError() {
        return error;
    }


    public String getLastLog() {
        return lastLog;
    }


}

File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SendDataResult.kt
package com.sierrawireless.avphone.task


class SendDataResult {
    var isError: Boolean = false
        private set
    var lastLog: String? = null
        private set
    var alarmLog: String? = null
        private set



    constructor(lastLog: String?, alarmLog: String?, error: Boolean) {
        this.isError = error
        this.lastLog = lastLog
        this.alarmLog = alarmLog
    }

    constructor(lastLog: String?, alarmLog: String?) {
        this.lastLog = lastLog
        this.isError = false
        this.alarmLog = alarmLog
    }
}
